src/foam/u2/tag/Input.js

```diff
@@ -32,6 +45,19 @@ foam.CLASS({
           this.warn('Set Input data to non-primitive:' + d);
           return o;
         }
+
+        if ( typeof this.validateValue === 'function' && o != undefined ) {
```

Comment:
validateValue can always equate a function if you make it a requirement on the prop.